systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -157,9 +156,9 @@
     public static final String OLM_OPERATOR_NAME = getOrDefault(OLM_OPERATOR_NAME_ENV, OLM_OPERATOR_NAME_DEFAULT);
     public static final String OLM_OPERATOR_DEPLOYMENT_NAME = getOrDefault(OLM_OPERATOR_DEPLOYMENT_NAME_ENV, OLM_OPERATOR_DEPLOYMENT_NAME_DEFAULT);
     public static final String OLM_SOURCE_NAME = getOrDefault(OLM_SOURCE_NAME_ENV, OLM_SOURCE_NAME_DEFAULT);
+    public static final String OLM_SOURCE_NAMESPACE = getOrDefault(OLM_SOURCE_NAMESPACE_ENV, null);
```

Comment:
Would it be better to use some `default` namespace as a default value?

Reply:
it use `OpenShift.OLM_SOURCE_NAMESPACE` in case the env war is empty, but I think I can use it directly here. The source of course could be deployed into any namespace

Reply:
Commited